app/src/test/java/nerd/tuxmobil/fahrplan/congress/schedule/TimeSegmentTest.java

```diff
@@ -4,33 +4,53 @@
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
+import java.util.TimeZone;
+
+import info.metadude.android.eventfahrplan.commons.SystemClock;
+
 import static org.assertj.core.api.Assertions.assertThat;
 
 @RunWith(JUnit4.class)
 public class TimeSegmentTest {
 
     @Test
-    public void getFormattedTextWith0() throws Exception {
+    public void getFormattedTextWith0() {
+        TimeZone.setDefault(TimeZone.getTimeZone("GMT+1"));
+
+        // minutesOfDay is considered UTC
```

Comment:
# Wording
Can you rephrase your comment to also use `GMT` instead of `UTC`? I would not mix both concepts to avoid confusion.

Reply:
GMT+1 can not be replaced with a UTC timezone string, since UTC is not a timezone. I only removed the comment.

Reply:
This is really confusing to me. I am not sure if GMT needs special handing because it never changes [because of daylight saving time](https://www.timeanddate.com/time/gmt-utc-time.html). If you like we talk about at some point. For now it look good for me. Feel free to resolve this.